models/articles.models.js

```diff
@@ -39,3 +39,59 @@ exports.selectAllArticles = () => {
       return res.rows;
     });
 };
+
+exports.selectArticleComments = (article_id) => {
+  return db
+    .query(
+      `SELECT *
+      FROM comments
+      WHERE article_id = $1
+        ORDER BY created_at DESC`,
+      [article_id]
+    )
+    .then((res) => {
+      return res.rows;
+    });
+};
+
+exports.checkIfArticleExists = (article_id) => {
+  return db
+    .query("SELECT * FROM articles WHERE article_id = $1", [article_id])
+    .then(({ rows }) => {
+      if (!rows.length)
+        return Promise.reject({ status: 404, msg: "Route Not Found" });
+    });
+};
+
+exports.checkArticleExists = (articleId) => {
+  return db
+    .query(
+      `
+      SELECT * FROM articles
+      WHERE
+      article_id = $1
+      `,
+      [articleId]
+    )
+    .then(({ rows }) => {
+      if (!rows.length) {
+        return Promise.reject({ status: 404, msg: "Article id invalid" });
+      }
+    });
+};
```

Comment:
You have 2 checkArticleExists functions here! Please get rid of the one you aren't using